packages/surf-distrotube/surf-distrotube.pacscript

```diff
@@ -0,0 +1,34 @@
+name="surf-distrotube"
+pkgname="surf"
+pkgver() {
+    git ls-remote "${url}" master | cut -f1
+}
```

Comment:
This needs to be written later as `url` isn't assigned yet, so it'll return nothing